src/web/PanGestureHandler.ts

```diff
@@ -11,6 +11,7 @@ import DraggingGestureHandler from './DraggingGestureHandler';
 import { isValidNumber, isnan, TEST_MIN_IF_NOT_NAN, VEC_LEN_SQ } from './utils';
 import State from '../State';
 
+import { GestureHandlerProperties } from '../types';
```

Comment:
Not used. Also, `../types` doesn't exist anymore.

Reply:
Probably my mistake, can be removed if is not used.